### PR TITLE
Dev account actions

### LINKED ISSUE #3165: [ioctl] build account actions command line into new ioctl
### What would you like to be added
 - Step 1: create `accountactions.go` and `accountactions_test.go` files in `./ioctl/newcmd/account`. 
 - Step 2: refactor  command  https://github.com/iotexproject/iotex-core/blob/master/ioctl/cmd/account/accountactions.go in `accountactions.go`
 - Step 3: add unit test in xx_test.go

#### Notes:
1. Use [client interface](https://github.com/iotexproject/iotex-core/blob/master/ioctl/client.go) to construct the Cobra command.
2. [Output package](https://github.com/iotexproject/iotex-core/blob/master/ioctl/output) is deprecated. `e.g.` `output.PrintError` could be replaced with `errors.Wrap`.
3. Global variables (config.ReadConfig  config.DefaultConfigFile etc.) have been wrapped into client interface. Please use [client interface](https://github.com/iotexproject/iotex-core/blob/master/ioctl/client.go) to access them if needed
4. The logic of new code should be similar to the old one.
5. In each unit test, the results should be checked whether they are as the expected. 

#### Example:
1. https://github.com/iotexproject/iotex-core/blob/master/ioctl/newcmd/account/accountinfo.go
2. https://github.com/iotexproject/iotex-core/blob/master/ioctl/newcmd/account/accountdelete.go
3. https://github.com/iotexproject/iotex-core/blob/master/ioctl/newcmd/account/accountcreateadd.go

### Why is this needed
Unit tests couldn't be added for the old ioctl command (./ioctl/cmd)

### PR DESCRIPTION
**Pull request by LuckyPigeon**
# Description

Build account actions command into ioctl/newcmd, since unit test couldn't add for old cmd, so needs for a new one.

Fixes #3165 

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] NewAccountActionsCmd

**Test Configuration**:

- Firmware version: Ubuntu 21.04
- Hardware:
- Toolchain:
- SDK:

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: ioctl/newcmd/account/accountactions.go [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+// Multi-language support
+var (
+	_actionsCmdShorts = map[config.Language]string{
+		config.English: "Show the list of actions for an account",
+		config.Chinese: "显示账户的操作列表",
+	}
+	_actionsCmdUses = map[config.Language]string{
+		config.English: "actions (ALIAS|ADDRESS)  [SKIP]",
+		config.Chinese: "actions (ALIAS|ADDRESS)  [SKIP]",
```
Comment (CoderZhi) [inline ioctl/newcmd/account/accountactions.go:50]: delete extra space

File: ioctl/newcmd/account/accountactions.go [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+			}
+			reqData := map[string]string{
+				"address": addr,
+				"offset":  fmt.Sprint(skip),
+			}
+			jsonData, err := json.Marshal(reqData)
+			if err != nil {
+				return errors.Wrap(err, "failed to pack in json")
+			}
+			resp, err := http.Post(client.Config().AnalyserEndpoint+"/api.ActionsService.GetActionsByAddress", "application/json",
```
Comment (CoderZhi) [inline ioctl/newcmd/account/accountactions.go:85]: create a new function `client.QueryAnalyser(input) (output, error)`

File: ioctl/newcmd/account/accountactions_test.go [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+	client.EXPECT().Config().Return(config.Config{
+		AnalyserEndpoint: "https://iotex-analyser-api-mainnet.chainanalytics.org",
+	}).AnyTimes()
+
+	t.Run("get account action", func(t *testing.T) {
+		accAddr := identityset.Address(28).String()
+		client.EXPECT().Address(gomock.Any()).Return(accAddr, nil)
+		cmd := NewAccountActionsCmd(client)
+		_, err := util.ExecuteCmd(cmd, accAddr, "0")
```
Comment (huof6829) [inline ioctl/newcmd/account/accountactions_test.go:46]: if no error,  it's better to  check  wether the account address of result is  identityset.Address(28)

File: ioctl/newcmd/account/accountactions_test.go [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+		require.NoError(err)
+	})
+
+	t.Run("empty offset", func(t *testing.T) {
+		accAddr := identityset.Address(28).String()
+		cmd := NewAccountActionsCmd(client)
+		_, err := util.ExecuteCmd(cmd, accAddr, "")
```
Comment (huof6829) [inline ioctl/newcmd/account/accountactions_test.go:53]: the same. thx

File: ioctl/newcmd/account/accountactions_test.go [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+		require.Error(err)
+	})
+
+	t.Run("empty address", func(t *testing.T) {
+		client.EXPECT().Address(gomock.Any()).Return("", nil)
+		cmd := NewAccountActionsCmd(client)
+		_, err := util.ExecuteCmd(cmd, "", "0")
```
Comment (huof6829) [inline ioctl/newcmd/account/accountactions_test.go:60]: the same , thx

File: ioctl/newcmd/account/accountactions.go (hunk 2 above)
Comment (huof6829) [reply ioctl/newcmd/account/accountactions.go]: ok

**Comment by LuckyPigeon** [comment]
@huof6829 Thanks!

**Comment by LuckyPigeon** [comment]
Don't we need to add some test data on these lines?
https://github.com/iotexproject/iotex-core/blob/71734f7ec72c1ec3acfdfb6073d38fbee68ab084/ioctl/newcmd/account/accountactions.go#L107-L114

Make them covered by tests.

**Comment by huof6829** [comment]
> @huof6829 Thanks!

You'r welcome

**Comment by huof6829** [comment]
> @huof6829 Don't we need to genereate some test data on these lines?
> 
> https://github.com/iotexproject/iotex-core/blob/71734f7ec72c1ec3acfdfb6073d38fbee68ab084/ioctl/newcmd/account/accountactions.go#L107-L114
> 
> Make them covered by tests.

These datas come from http response.  I found response data is null, so cannot cover.  You can try to mock data by QueryAnalyser, this is a bit complicated.

**Comment by huof6829** [comment]
@LuckyPigeon  Thanks for adding test datas!

**Review by huof6829 — APPROVED** [review]

**Review by Liuhaai — APPROVED** [review]

**Pull request merged**